packages/build/index.ejs

```diff
@@ -1,9 +1,8 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="referrer" content="origin" />
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
You need to bring it back. This is to prevent leaking the referrer information when clicking on the URL links. https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noreferrer

This is separate from REST calls.

Reply:
revised